src/07-projects/milestones.apib

```diff
@@ -104,6 +117,19 @@ Get details for a single milestone.
                 + (object)
                     + type: `milestone` (string)
                     + id: `b71ff11f-074e-0705-9450-589342af4f4f` (string)
+            + actuals (object) - Only accessible for administrators of the project this milestone belongs to
+                + `billable_amount` (Money)
+                + costs (Money)
+                + result (Money) - The billable amount minus the costs, so the profit or loss
+            + budget (object) - Only accessible for administrators of the project this milestone belongs to
+                + provided (Money)
+                + spent (object)
```

Comment:
Spent is exposed as `Money` object on `milestones.info`

Reply:
Yes, but here it's divided into total, time, and materials. So a regular object